### PR TITLE
transitions to dcos/examples

### PR DESCRIPTION
**Pull request by mhausenblas**
## What type of changes does your doc introduce?
- [x] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [x] Blocker
- [ ] High
- [ ] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

Please merge ASAP so that we can cut over to https://github.com/dcos/examples

### CODE REVIEW

File: 1.8/usage/tutorials/index.md [hunk 1]
```diff
@@ -2,4 +2,39 @@
 post_title: Tutorials
 menu_order: 90 
 ---
-This is a collection of tutorials about using DC/OS. Learn how to run services and operate services in production.
+
+This is the DC/OS tutorials section, covering how to use DC/OS components as well as stateless and stateful DC/OS services. Typically, the services discussed here are available and installable from the [Universe](https://github.com/mesosphere/universe).
+
+The left-hand side menu lists the available basic tutorials, such as how to create and run a service. Further, there are a number of examples of how to use DC/OS services such as Spark or WordPress, which are available via the [dcos/examples](https://github.com/dcos/examples/) GitHub repo.
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/index.md]: Do we want to explain here why we have separate locations for these tutorials (e.g. examples located in GH repo are tested/validated?). In looking what we have now - it just seems like a formality that the content is displayed in one format or another. Also, explaining a left-hand menu seems irrelevant since the links are also repeated in the body.  

CC: @williamsaaron

File: 1.8/usage/tutorials/index.md [hunk 2]
```diff
@@ -2,4 +2,39 @@
...
+
+Continuous Integration/Continuous Deployment:
```
Comment (joel-hamill) [inline 1.8/usage/tutorials/index.md]: Continuous Integration and Continuous Deployment

File: 1.8/usage/tutorials/index.md (hunk 1 above)
Comment (joel-hamill) [inline 1.8/usage/tutorials/index.md]: `The tutorials that are located in [dcos/examples](https://github.com/dcos/examples/) GitHub repo are the most up-to-date. These tutorials are guaranteed to work the latest [versions](https://dcos.io/releases/) of DC/OS.`

File: 1.8/usage/tutorials/index.md (hunk 2 above)
Comment (mhausenblas) [inline 1.8/usage/tutorials/index.md]: Fine with me
